### PR TITLE
{math}[goolf/1.5.16, intel/2018a] SCOTCH 6.0.4 (REVIEW)

### PR DESCRIPTION
**Pull request by SimonPinches**

### CODE REVIEW

File: easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.4-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+homepage = 'http://gforge.inria.fr/projects/scotch/'
+description = """Software package and libraries for sequential and parallel graph partitioning,
+static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://gforge.inria.fr/frs/download.php/file/34618/']
+sources = ['%(namelower)s_%(version)s.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.4-intel-2018a.eb]: @SimonPinches `SHA256` checksum is missing here?

Comment (SimonPinches) [reply easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.4-intel-2018a.eb]: Oops, added now.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2040.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4b264add3a429f077300b21b77b25854 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2523.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c0b5a59da7228f7f834a15a91f3ae21c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**